content/blog/get-started-with-application-performance-monitoring-tools-overview.md

```diff
@@ -0,0 +1,235 @@
+
+### **Apache SkyWalking**
+
+![](/img/skywalking.png)
+
+[Apache SkyWalking](https://skywalking.apache.org/) is an open source APM tool with capabilities for monitoring, tracing and diagnosing distributed system. It’s especially designed for microservices, cloud native and container-based architectures. 
+
+Key features:
+
+* provide metrics analysis of services, service instances and endpoints with distributed tracing, log collecting and metrics collecting and customization;
+* support root cause analysis with profiling the code on the runtime by in-process agent, _eBPF_ profiler and network profiler;
+* provide dependency analysis of service instances and endpoins; 
+* support service topology map analysis;
+* detect slow services and endpoints and provide performance optimization;
+* detect slow _SQL_ statement for database performance monitoring;
+* provide message queue performance and consuming latency monitoring;
+* start tracing from browser for browser performance monitoring;
+* support infrastructure monitoring for Kubernetes and Linux;
+* support alerting using rules in both observability analysis language and metric analysis language;
+
+Apache SkyWalking provides a list of agents to be used for building *Java*, *.NET Core*, *PHP*, *Node.js*, *Golang*, *LUA*, *Rust* and *C++* apps. It supports to integrate and collect data from multiple sources, including _Prometheus_, _OpenTelemetry_ and _Zabbix_ for metrics and logs, _Zipkin_ for traces. It provides tracing, metrics analysis, alerting, service mesh observability and visualization. 
+
+Apache SkyWalking is an open source tool with *21K* GitHub stars and *6K* GitHub forks. In [StackShare community](https://stackshare.io/apache-skywalking#stacks.), Apache SkyWalking does not yet have many approval, only being  mentioned in 12 developer stacks. However, Apache SkyWalking has more than *600* contributors on GitHub and thousands of contributions every year. All the agents for application instrumentation have been actively maintained. 
+
+Apache SkyWalking is the first open source project that initialized and implemented an [Envoy Access Log Service (ALS)](https://www.envoyproxy.io/docs/envoy/v1.18.2/api-v2/service/accesslog/v2/als.proto) based solution to provide observability on the service mesh, no matter the architecture or language. Since service mesh provides full control of _RPC_, this observation solution is much easier to be added without language specific technology. With this solution, users could get the application service topology map, metrics graph, request details and error message with a very nice visualization. This integration solution can be extremely important for monitoring and visualizing application that consists of many microservices running across on-premises, cloud-based or hybrid environments. 
```

Comment:
2nd sentence, add a hyphen between "language" and "specific" so it says "... language-specific technology."

3rd sentence, change "users could get" to "users **can** get".

Last sentence, change to "for monitoring and visualizing applicatio**ns** that **consist** of..."